packages/uui-textarea/lib/uui-textarea.element.ts

```diff
@@ -305,8 +226,9 @@ export class UUITextareaElement extends FormControlMixin(LitElement) {
 
     input.style.height = 'auto';
 
-    if (input.scrollHeight > input.clientHeight) {
-      input.style.height = input.scrollHeight + 'px';
+    // Note: Add + 2 because of the border top+bottom 1px each
+    if (input.scrollHeight + 2 > input.clientHeight) {
```

Comment:
I'm just reading the code now, but if I'm not mistaken this field can never become less tall with this code.

I would assume you need to make an else case (Maybe with an additional test, to make sure it only happens when the scrollHeight is more than 1 or 3 pixels smaller than input.clientHeight.

Something like this:

`
... else if (input.scrollHeight + 2 < input.clientHeight) {
      input.style.height = '';// <-- setting a style to empty string resets the value.
`